.changes/next-release/enhancement-cloudfront-1337.json

```diff
@@ -0,0 +1,5 @@
+{
+  "type": "enhancement",
+  "category": "``cloudfront``",
+  "description": "Replace cryptographic functions from ``cryptography`` with ``awscrt`` for the ``sign`` command. Fixes https"
```

Comment:
Looks like a fragment for the link?

Reply:
Yikes, not sure what happened there. Thanks!